test/parallel/test-trace-event.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const assert = require('assert');
+const cp = require('child_process');
+const common = require('../common.js');
```

Comment:
For tests, I believe `common` is supposed to be the first line (after `'use strict'`), see nodejs/node#7786.